server/index.js

```diff
@@ -11,12 +11,36 @@ let port = process.env.PORT || 5000;
 app.use(cors());
 app.use(express.json());
 
+const session = require("express-session");
+const cookieParser = require("cookie-parser");
+
+const passport = require("passport");
+require("./config/passport-googleAuth-strategy");
+
+app.use(express.urlencoded({ extended: false }));
+app.use(cookieParser());
+
+app.use(
+  session({
+    name: "SSID",
+
+    secret: process.env.SECRET_KEY,
+    saveUninitialized: false,
+    resave: false,
+    cookie: {
+      maxAge: 1000 * 60 * 100,
+    },
+  }),
+);
+app.use(passport.initialize());
+app.use(passport.session());
+
 app.get("/", (req, res) => {
   res.send("HELLO FROM HOME");
 });
 
 //* user routes
-app.use("/user", require("./routes/User"));
+app.use("/user", require("./routes/user"));
```

Comment:
Please revert this back to `./routes/User` as we have the file name as `User.js` having small letter can sometimes give rise to bugs.